ee/packages/omnichannel-services/src/OmnichannelTranscript.ts

```diff
@@ -232,8 +232,20 @@ export class OmnichannelTranscript extends ServiceClass implements IOmnichannelT
 					continue;
 				}
 
-				const fileBuffer = await uploadService.getFileBuffer({ file: uploadedFile });
-				files.push({ name: file.name, buffer: fileBuffer, extension: uploadedFile.extension });
+				try {
+					const fileBuffer = await uploadService.getFileBuffer({ file: uploadedFile });
+					files.push({ name: file.name, buffer: fileBuffer, extension: uploadedFile.extension });
+				} catch (e: unknown) {
+					this.log.error(`Failed to get file ${file._id}`, e);
+					// Push empty buffer so parser processes this as "unsupported file"
```

Comment:
I didn't find where this is being handled, can you show me where this is being handled as an unsupported file?

Reply:
That happens inside `pdf-worker`. When a `null` buffer is provided, it'll render an "Unsupported file" box on the transcript.